### PR TITLE
Fixes #2376 - Add Swarnava

### LINKED ISSUE #2376: add @swarnava to team section

### PR DESCRIPTION
**Pull request by swarnava**

### CODE REVIEW

**Comment by zoepage** [comment]
Gonna review this in a bit :)

File: webcompat/templates/contributors.html [hunk 1]
```diff
@@ -143,6 +143,26 @@ <h4 class="headline-3">Romania</h4>
         </div>
       </article>
 
+      <!-- Swarnava Sengupta -->
```
Comment (zoepage) [inline webcompat/templates/contributors.html]: Please add your name per alphabetic order as the list is organized like this. thanks!

**Comment by zoepage** [comment]
I just realized there was no issue created: #2376.

Could you also please change the commit message from `Add Swarnava` to `Issue #2376 - add Swarnava`.

If you have questions, please let me know. Thanks!

**Comment by swarnava** [comment]
@zoepage Do I need to rebase?

**Comment by zoepage** [comment]
no :)
Following should work:

```
git commit --amend 
// change message
git push -f origin swarnava:contributor // or what your origin indicator is :)
```

**Comment by zoepage** [comment]
Oh, I just saw that you have already pushed the commit...
Another way to fix it is:

```
git log
// check out which commit was before your first one
// copy that commit number
git reset --soft xxxnumber
git add webcompat/templates/contributors.html
git commit -m "Issue #2376 - add Swarnava"
git push -f origin swarnava:contributor // or what your origin indicator is :)
```

**Comment by swarnava** [comment]
creating a new pr 😓

**Comment by swarnava** [comment]
#2377

**Comment by zoepage** [comment]
closing in favor of #2377

**Pull request closed without merging**